website/src/footer.tsx

```diff
@@ -1,15 +1,29 @@
 import React from "react"
-import { Link } from "src/components"
+import { CreativeCommons, Link } from "src/components"
 import { usePageContext } from "src/renderer/PageShell"
 import { wordpressUrl } from "src/theme.css"
 import * as css from "./footer.css"
+import { CreativeCommonsBy } from "./components/attributions/creative-commons"
 
 /** University affiliation, related navigation links, and contact info.  */
 const Footer = () => {
   const { buildDate } = usePageContext()
   return (
     <footer className={css.footer}>
       <div className={css.dark}>
+        <span className={css.content} style={{ display: "block" }}>
+            <CreativeCommonsBy 
+              title="The Digital Archive of Indigenous Language Persistence (DAILP)"
+              authors={
```

Comment:
Would these authors possibly change at all in the future? I'm wondering if we would want to store this information in our database rather than hardcoding it here so it can be more flexible to change in the future

Reply:
No we definitely do want it in the DB. I can make those changes in a new issue and PR maybe? I want to at least get this scaffolding online sooner than later